### DISCUSSION TITLE
Can't install  HomeKit firmware

### DISCUSSION THREAD

**Discussion by risk235** [ratgdo/homekit-ratgdo#256] [Q&A]
When I try to install the HomeKit firmware it tells me your esp32 board is not supported

**Comment by donavanbecker** [comment]
You need this repo: https://github.com/ratgdo/homekit-ratgdo32 which the firmware is currently being developed for.

**Reply by risk235** [reply]
thank you for the response. I'm new to this. How do I install the firmware from there? I downloaded the zip file

**Reply by donavanbecker** [reply]
As I stated before, it is being developed. You could install the https://github.com/ratgdo/esphome-ratgdo firmware or wait for the native HomeKit firmware to come out.

**Comment by dkerr64** [comment]
I just published working firmware are the ratgdo32 link.  I have not tested the browser install yet, but you could try it.

**Reply by risk235** [reply]
Thank you. I will try web browser when I get home. If by chance it doesn’t work yet how do I go about to install it manually

**Comment by dkerr64** [comment]
Well... I tested this morning and the browser install did not work. Which means at the moment it will only install from a full development environment.  But I'm going to investigate why install is failing later today so hopefully will have a fix.  I'm going to close this issue and ask that you monitor this one... https://github.com/ratgdo/homekit-ratgdo32/issues/14

Thanks.
